django_irods/storage.py

```diff
@@ -436,7 +436,13 @@ def size(self, name):
             raise ValidationError(
                 "{} cannot be found in iRODS to retrieve " "file size".format(name)
             )
-        return int(float(stdout))
+        # remove potential '\n' from stdout
+        size_string = stdout.strip("0\n").replace("\n", "")
+        try:
+            ret = int(float(size_string))
+            return ret
+        except ValueError:
+            return 0
```

Comment:
In the code base in many places we have assumed a file size of 0 means file is missing from irods. May be we have to re-evaluate that assumption in a later release.